django_freeradius/api/views.py

```diff
@@ -28,7 +28,10 @@ def authenticate(self, request):
         if request.META.get('HTTP_AUTHORIZATION', False):
             headers = request.META.get('HTTP_AUTHORIZATION').split(',')
             for header in headers:
-                token = header.split(' ')[1]
+                try:
```

Comment:
I wouldn't use a `try`/`except` here, there are more readable solutions to this. I'd probably do something like:
```suggestion
tokens = header.split(' ')
if len(tokens) > 1:
    token = tokens[1]
else:
    token = header[6:]
```

Reply:
I think we should be avoiding the usage of exceptions as flow control, as much as possible.

Reply:
Line 31 would still return a IndexError in this case as there is no space i think it is necessary to catch that error so to be able to pass the test tho i might be wrong. What are your thoughts? @R9295 @nemesisdesign @AnonGuy

Reply:
It won't raise an IndexError, even if there isn't a space.

Reply:
What are your thoughts on this? @R9295 @nemesisdesign

Reply:
Say `header` has the value `"foobar"`, `header.split(" ")` won't raise an IndexError, it will return the list `["foobar"]`.

Reply:
(There's no need to delete your messages)

Reply:
Yeah you are right i won't raise any error.

Reply:
@AnonGuy using try/except in python is perfectly acceptable and actually recommended unless the exceptional case is common

Reply:
@cking100 I will review tomorrow